resources/keepalived-docker/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ubuntu
-copy sample.keepalived.conf /etc/keepalived/keepalived.conf
+ENV DEBIAN_FRONTEND=noninteractive
+COPY sample.keepalived.conf /etc/keepalived/keepalived.conf
 COPY  manage-keepalived.sh manage-keepalived.sh
 RUN apt update -y && apt install keepalived -y
-ENTRYPOINT /bin/bash manage-keepalived.sh && tail -f /dev/null
+ENTRYPOINT exec /bin/bash manage-keepalived.sh
```

Comment:
I believe you can write this as `ENTRYPOINT ["/bin/bash", "manage-keepalived.sh"]` to avoid starting two shells.

Reply:
good point, fixed, thank you